views/tenants/index.erb

```diff
@@ -1 +1,11 @@
-Listing tenants for apartment #3
+<h2>Apartment at <%= @apartment.address %></h2>
+<h3>Tenants</h3>
+<ul>
+  <% tenants = @aparment.tenants %>
```

Comment:
Instead of having a line of Ruby here that doesn't really affect the HTML of the page, I'd put this in your controller:

```
@tenants = @apartment.tenants
```